### PR TITLE
Add TOML version where YAML version also mentioned for dvc#7253

### PR DESCRIPTION
**Pull request by syntapy**
Slight edit to be made per https://github.com/iterative/dvc/pull/7468

> You may disregard these recommendations if you used the **Edit on GitHub** button from dvc.org to improve a doc in place.

❗ Please read the guidelines in the [Contributing to the Documentation](https://dvc.org/doc/user-guide/contributing/docs) list if you make any substantial changes to the documentation or JS engine.

🐛 Please make sure to mention `Fix #issue` (if applicable) in the description of the PR. This causes GitHub to close it automatically when the PR is merged.

Please choose to [allow us](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork) to edit your branch when creating the PR.

Thank you for the contribution - we'll try to review it as soon as possible. 🙏

### CODE REVIEW

**Comment by jorgeorpinel** [comment]
Draft-ed since https://github.com/iterative/dvc/pull/7468 is still a draft. Please mark ready to review when the core PR is approved. Thanks

**Comment by jorgeorpinel** [comment]
Hi. Seems like https://github.com/iterative/dvc/pull/7468 is inactive. Should we keep this still? Thanks

**Comment by syntapy** [comment]
@jorgeorpinel I've been (slowly) working on an [upstream pr](https://github.com/sdispater/tomlkit/pull/187) to finish the one in dvc, as per [comment](https://github.com/iterative/dvc/pull/7468#discussion_r839196940). So yeah, I'm still on it!

**Comment by syntapy** [comment]
@jorgeorpinel ready to be looked at!

**Review by daavoo — APPROVED** [review]

File: content/docs/command-reference/params/index.md [hunk 1]
```diff
@@ -47,7 +47,7 @@ dependency (e.g. a config file), and any change in it invalidates all of them
 (see `dvc status`), causing unnecessary re-executions upon `dvc repro`.
 
 The default **parameters file** name is `params.yaml`, but any other YAML 1.2,
-JSON, TOML, or [Python](#examples-python-parameters-file) files can be used
+JSON, TOML 1.0, or [Python](#examples-python-parameters-file) files can be used
 additionally (listed under `params:` as shown in the sample above). These files
```
Comment (jorgeorpinel) [inline content/docs/command-reference/params/index.md:51]: Thanks! There's another instance we missed in https://dvc.org/doc/command-reference/metrics#supported-file-formats

Comment (daavoo) [reply content/docs/command-reference/params/index.md:51]: https://github.com/iterative/dvc.org/pull/3871

**Pull request merged**